private/buf/buflsp/symbol_test.go

```diff
@@ -0,0 +1,58 @@
+package buflsp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_formatComment(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
+			name:     "Single-line comment",
+			input:    "// This is a single-line comment",
+			expected: " This is a single-line comment",
+		},
+		{
+			name:     "Multi-line comment",
+			input:    "/*\n This is a\n multi-line comment\n */",
+			expected: "This is a\nmulti-line comment",
+		},
+		{
+			name:     "Multi-line comment with mixed indentation",
+			input:    "/*\n  * First line\n  * - Second line\n  *   - Third line\n */",
```

Comment:
This test case is incorrect. You should not trim asterisks if the comment does not begin with `/**`. Alternatively, you could instead only trim asterisks if *every* line starts with one.